README.md

```diff
@@ -1 +1,42 @@
-# Hello_Microverse_Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] Can you please write the actual name of the project, Hello Microverse. This will make you repository look professional as it is the first thing anyone viewing your repo sees.